src/target/riscv/riscv.c

```diff
@@ -506,6 +506,14 @@ static int find_trigger(struct target *target, int type, bool chained, unsigned
 	return ERROR_FAIL;
 }
 
+static int find_trigger(struct target *target, unsigned int type, bool chained,
+						unsigned int *idx)
+{
+	if (idx)
+		*idx = 0;
+	return find_next_trigger(target, type, chained, idx);
+}
+
```

Comment:
This feels like unnecessary complexity. find_trigger() is always called with a non-zero idx pointer. It is better to modify all the call sites to force the initial value to 0 than adding this tiny function.

Reply:
Addressed.